dists/install.sh

```diff
@@ -110,7 +118,7 @@ main() {
   tmpdir=$(mktemp -d /tmp/scalingo_cli_XXX)
   trap "clean_install ${tmpdir}" EXIT
 
-  version=$(curl --silent https://cli-dl.scalingo.com/version | tr --delete ' \t\n')
+  version=$(curl --silent https://cli-dl.scalingo.com/version | tr -d ' \t\n')
```

Comment:
question(non blocking): Maybe add a comment everywhere you use short option to explain the reason? As the Scalingo good practice is to use long options, we will maybe be tempted to update these options again in the future.